MDISforLinux/DRIVERS/BBIS/SMB2BB/DRIVER/NATIVE/bb_smb2.c

```diff
@@ -983,8 +990,6 @@ static int oss_smb2_remove(struct i2c_client *client)
     kfree(data);
 
     /* -- data invalid now -- */
-
-    return err;
```

Comment:
The returning value should be also protected with kernel version check as for kernel version < 6.1, the function oss_smb2_remove returns an integer.

Reply:
This return was a bit useless as it was always returning the initial value of err = 0; I'll just leave it as a `return 0`